examples/example_otap.py

```diff
@@ -30,7 +30,8 @@
                                         "propagate_only",
                                         "immediately",
                                         "delayed",
-                                        "update_delay"])
+                                        "update_delay",
+                                        "remove"])
```

Comment:
The correct term would be more "disable" or "no_otap"
"Remove" sounds like you are erasing the scratchpad from devices, but it is not the case. Only action is set to no_otap. The scratchpad remains in the nodes.